### ISSUE TITLE
Module parse failed: Unexpected token (20:8) Failed to compile.

### ISSUE THREAD

**Issue by TonyCrane** [rossning92/movy#1]
I tried to run it according to the README, but this exception was thrown:
```js
ERROR in ...\npm\node_modules\movy\src\movy.ts 20:8
Module parse failed: Unexpected token (20:8)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| import TextMesh from "./objects/TextMesh";
|
> declare class CCapture {
|   constructor(params: any);
| }

webpack 5.18.0 compiled with 1 error in 2654 ms
i ｢wdm｣: Failed to compile.
```
I am not familiar with JavaScript, hope to get help. :)

**Comment by StageGuard** [comment]
I got the same problem(

**Comment by rossning92** [comment]
My bad, fixed in v0.0.4. You could try `npm i -g movy` again to upgrade to the latest version.
